escaper.go

```diff
@@ -20,7 +20,7 @@ func EscapeArg(arg driver.NamedValue) (string, error) {
 	case bool:
 		return fmt.Sprintf("%v", v), nil
 	case string:
-		return fmt.Sprintf("'%v'", strings.ReplaceAll(v, "'", "''")), nil
```

Comment:
I see. so previously we were embracing string in a quotation mark. interesting find @mattdeekay